firecrown/ccl/two_point.py

```diff
@@ -28,10 +29,18 @@ def parse_config(analysis):
     if 'likelihood' in analysis:
         new_keys['likelihood'] = _parse_likelihood(analysis['likelihood'])
 
-    # read data if there is a sacc file
+    # read data if there is a sacc file or read object
     if 'sacc_file' in analysis:
-        sacc_data = sacc.Sacc.load_fits(
-            os.path.expanduser(os.path.expandvars(analysis['sacc_file'])))
+        warnings.warn("sacc_file is deprecated, use sacc_data instead",
+                      DeprecationWarning)
+        analysis['sacc_data'] = analysis['sacc_file']
```

Comment:
This warning is nice but not needed. I would rather us make the hard change now (i.e. no longer support this) and go through and adjust all of the configs we have. I don't want to carry this around forever.